ci/live-cluster-sanity.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+set -e
+cd "$(dirname "$0")/.."
+
+source ci/_
+source ci/rust-version.sh stable
+
+escaped_branch=$(echo "$BUILDKITE_BRANCH" | tr -c "[:alnum:]" - | sed -r "s#(^-*|-*head-*|-*$)##g")
+instance_prefix="testnet-live-sanity-$escaped_branch"
+# ensure to delete leftover cluster
+./net/gce.sh delete -p "$instance_prefix" || true
+# only bootstrap, no normal validator
+./net/gce.sh create -p "$instance_prefix" -n 0
```

Comment:
Let's ensure the instances are shut down promptly if something goes wrong:
```suggestion
./net/gce.sh create -p "$instance_prefix" -n 0 --self-destruct-hours 1
```

Reply:
thanks for tipping this nice option. I didn't know it.